shortcuts.yaml

```diff
@@ -1,26 +1,29 @@
 short options:
```

Comment:
What would help to verify the consistency but also to ensure it in the future is to have a paragraph explaining the thought process behind selecting specific shorts for specific options. Some choices seem logical but some seem arbitrary so it would help to be able to verify why you've made specific choices.

Reply:
I started a document in the other branch. We can prettify it? What do you think?

https://github.com/VelinGeorgiev/office365-cli/commit/7dc53fd3f0f11000a7305b3d746651233eb8e4a1#diff-795cb50144940c2309c7fb6185beb678

We can remove some that do not make any sense . The list you are seeing is autogenerated with what we have so far and some that I shuffled to make it more consistent.

Reply:
Nice! That is helpful. I'll go through it and leave comments if any.